commodore/catalog.py

```diff
@@ -175,6 +175,24 @@ def _kapitan_029_030_difffunc(
     before_text: str, after_text: str, fromfile: str = "", tofile: str = ""
 ) -> tuple[Iterable[str], bool]:
 
+    [diff, _] = _ignore_non_semantic_whitespace_difffunc(
+        before_text, after_text, fromfile, tofile
+    )
+
+    diff_lines = list(diff)
+
```

Comment:
This is a noop, since we already call `list(diff_lines)` in `_ignore_non_semantic_whitespace_difffunc`

```suggestion
```

Reply:
Mypy is unhappy without

Reply:
ah, that would be because `_ignore_non_semantic_whitespace_difffunc()` indicates that it returns an `Iterable[str]`. Since we know that we return a list there, I'd just change that to `list[str]` in the return type.